### ISSUE TITLE
Creating a default community health file

### ISSUE THREAD

**Issue by linuzb** [alipay/container-observability-service#30]
[About default community health files](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#about-default-community-health-files)

**Comment by linuzb** [comment]
#29

**Comment by larryck** [comment]
Yeah, that's might be one of the top priority jobs of us right now. I will submit a draft of CONTRIBUTING.md and GOVERNANCE.md first.
